integration-tests/pipelines/konflux-e2e-tests.yaml

```diff
@@ -0,0 +1,226 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Pipeline
+metadata:
+  name: konflux-e2e-tests
+spec:
+  description: |-
+    This pipeline automates the process of running end-to-end tests for Konflux
+    using a ROSA (Red Hat OpenShift Service on AWS) cluster. The pipeline provisions
+    the ROSA cluster, installs Konflux using the infra-deployments, runs the image-controller tests, collects artifacts,
+    and finally deprovisions the ROSA cluster.
+  params:
```

Comment:
How can I change some params for the pipeline?

Reply:
we need to change it via integration tests